charts/ambassador/ci/update_chart_changelog.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I'm surprised that this isn't just the toybox Python script? No big deal reconciling that later though.

Reply:
they toybox one does similar things but is also in python and not bash (bc im masochistic apparently)
i put this in here before i knew toybox things were moving/emissary was even happening so that OSS peeps could run this too. yeah, i think lets reconcile later (soon-ish, but not now?)

Reply:
Soonish but not now sounds fine, yeah. 🙂